### PR TITLE
Update source migration page and updated videos

### LINKED ISSUE #301: FAQ: Mangadex Video Guide update
The website's mangadex video guide is outdated

### LINKED ISSUE #305: Guides: Video Guide Update
Videos in the Guides pages are out of date

### LINKED ISSUE #311: Guides: Add Migration Video to Stable Migration

### PR DESCRIPTION
**Pull request by Seishirou101**
closes #301 
closes #305 
closes #311

### CODE REVIEW

**Comment by Soitora** [comment]
The "closed #x" needs to be a keyword in front of every and not just the first, also I *think* it needs to be in the description and not the title

Title should preferably state the changes in a summary

**Comment by Seishirou101** [comment]
updated title and description

File: src/help/faq/README.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ This error can be caused by a variety of reasons, all to do with storage.
 
 #### Why are there no results when searching?
 ::: aside-guide
-[<MaterialIcon icon="videocam"/> Click for video guide](/help/faq/assets/MangaDex-NoResults.webm)
+[<MaterialIcon icon-name="videocam"/> Click for video guide](/help/faq/assets/md_noresults.mp4)
```
Comment (GHOSTBEAR) [inline src/help/faq/README.md:255]: The icon doesn't show due to the attribute being renamed to only `icon` from `icon-name`. The correct way to use **MaterialIcon** would be `<MaterialIcon icon="videocam"/>`

Comment (Seishirou101) [reply src/help/faq/README.md]: fixed. hopefully, I got all the references of it after I rebased

File: src/help/guides/source-migration.md [hunk 2]
```diff
@@ -8,14 +8,174 @@ lang: en-US
...
-	> **Tachiyomi** will do a global search of all the sources you have installed and enabled. If you don't find the manga you want, try searching for other titles it might go under.
-1. Once you find the manga you want to migrate, select it and then choose which data you want to transfer over, and you're done.
+There are two ways to migrate, you can do it by it by manga or by source.
+
+::: guide Stable/Preview Migration 
+[<MaterialIcon icon="videocam"/> Click for video guide](/help/guides/source-migration/assets/migrate_preview.mp4)
+
+By **Manga**:
+1. Tap into a **Manga** you would like to migrate.
+1. Go to <MaterialIcon iconName="more_vert"/> and tap **Migrate**.
```
Comment (GHOSTBEAR) [inline src/help/guides/source-migration.md:18]: There are quite a few **MaterialIcon** tags that use the attribute `iconName` in this file, change these attributes to `icon`. If you build the project locally with `npm run serve` you can see that there are errors in the DevTools console when you visit this guide

Comment (Seishirou101) [reply src/help/guides/source-migration.md]: done.

> you can see that there are errors in the DevTools console when you visit this guide

Thanks for this tip. This helped me a lot.

File: src/help/guides/source-migration.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ There are two ways to migrate, you can do it by it by manga or by source.
 
 By **Manga**:
 1. Tap into a **Manga** you would like to migrate.
-1. Go to <MaterialIcon iconName="more_vert"/> and tap **Migrate**.
+1. Go to <MaterialIcon icon="more_vert"/> and tap **Migrate**.
```
Comment (Soitora) [inline src/help/guides/source-migration.md:18]:
```suggestion
1. Go to <Navigation item="more" /> and tap **Migrate**.
```
Stuff like this (including "Extensions") should be used as a Navigation item, meaning if anything changes in the future you only need to update the main file.

File: src/help/guides/source-migration.md [hunk 4]
```diff
@@ -50,22 +50,22 @@ All [Forks](/forks/) but Neko have source migration of multiple manga instead of
 ::: guide TachiyomiJ2K Migration
 [<MaterialIcon icon="videocam"/> Click for video guide](/help/guides/source-migration/assets/migrate_j2k.mp4)
```
Comment (Soitora) [inline src/help/guides/source-migration.md:51]:
```suggestion
::: aside-guide
[<MaterialIcon icon="videocam"/> Click for video guide](/help/guides/source-migration/assets/migrate_j2k.mp4)
:::
```
These should be as an aside-guide, when including a three-colon object in another though you need to increase the main one, so `::: guide TachiyomiJ2K Migration` becomes `:::: guide TachiyomiJ2K Migration`, including the ending `:::` to `::::`

**Review by Soitora — CHANGES_REQUESTED** [review]

**Comment by Soitora** [comment]
Somehow my review added on to my other review (even though it said only 2 reviewed), which caused oddities. Hope you can still see what I mean

File: src/help/guides/source-migration.md [hunk 5]
```diff
@@ -8,14 +8,174 @@ lang: en-US
...
-::: guide
-1. Go to <Navigation item="more"/> and tap <Navigation item="source_migration"/>.
-1. Select the **Source** you're migrating from.
-1. Find and select the title that you want to migrate.
-	> **Tachiyomi** will do a global search of all the sources you have installed and enabled. If you don't find the manga you want, try searching for other titles it might go under.
-1. Once you find the manga you want to migrate, select it and then choose which data you want to transfer over, and you're done.
+There are two ways to migrate, you can do it by it by manga or by source.
+
+::: guide Stable/Preview Migration 
+[<MaterialIcon icon="videocam"/> Click for video guide](/help/guides/source-migration/assets/migrate_preview.mp4)
```
[the remark this answers is not in the document]
Comment (Seishirou101) [reply src/help/guides/source-migration.md]: Will do, thanks

File: src/help/guides/source-migration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Seishirou101) [reply src/help/guides/source-migration.md]: changed with the addition of the Navigation item overflow.

**Comment by Seishirou101** [comment]
need to split videos for changes soi made

**Comment by Seishirou101** [comment]
videos have been updated

**Comment by Soitora** [comment]
I'll accept it for now as it looks fine, if anyone catches anything wrong do make an issue about it or contact either of us.

**Review by Soitora — APPROVED** [review]

**Pull request merged**